EIPS/eip-4762.md

```diff
@@ -52,23 +52,23 @@ process this access event:
 (address, 0, BASIC_DATA_LEAF_KEY)
 ```
 
-Note: a non-value-bearing `SELFDESTRUCT` or `*CALL`, targetting a precompile, will not cause the `BASIC_DATA_LEAF_KEY` to be added to the witness.
+Note: a non-value-bearing `SELFDESTRUCT` or `*CALL`, targetting a precompile or a system contract, will not cause the `BASIC_DATA_LEAF_KEY` to be added to the witness.
```

Comment:
but the gas costs will be charged because of https://github.com/ethereum/EIPs/pull/8707/files#diff-cd2aaa222faeb6081a41605386a509b3c4839285cd15f1b0ebe009318707914eR44 right?

Reply:
if so, I would add a sentence to confirm it, so that people are not confused.

Reply:
I think it should be 2929 cold or hot account access cost (and not cold 6800 witness cost) for system contracts

and for precompiles it would be warm access 2929 access cost

Reply:
clarified that leaf won't be accessed for system contract as well